test/parallel/test-http-content-length-mismatch.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+function shouldThrowOnMoreBytes() {
+  const server = http.createServer(common.mustCall((req, res) => {
+    res.setHeader('Content-Length', 5);
+    assert.throws(() => {
+      res.write('hello');
```

Comment:
can you move this line outside the throws check?

Reply:
Sure thing, just curious why are we doing this though? to reduce nested closures?